front-end/src/components/modal/download/DownloadModal.jsx

```diff
@@ -38,8 +38,7 @@ export const DownloadModal = ({ setModal }) => {
       
       // Add each selected file to the zip archive
       for (const file of selectedFiles) {
-        const response = await ApiUtil.getFile(file.key);
-        const blob = await response.blob();
+        const blob = await ApiUtil.getFile(file.key);
```

Comment:
Same curiosity about this working if you changed something to fix this or if this is an unrelated improvement

Reply:
ApiUtils now returns the parsed response via 'response.json()' or similar.